src/commands/question.js

```diff
@@ -0,0 +1,16 @@
+import config from '../../config';
+import { getRandom } from '../util';
+const errorResponses = ['I couldn\'t find anything.',
+'Yuka-chan is smarter than me, she would know.',
+'Only the Fox God knows.'];
+
+class Command {
+  constructor() {
+    this.aliases = ['question', 'question\,'];
```

Comment:
I think these aliases should be added to query.js instead. Think of a different alias for this, maybe answer, maybe qq, something like that.